build.gradle

```diff
@@ -40,25 +54,12 @@ test {
     }
 }
 
-buildscript {
-    repositories {
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
-    }
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-        classpath 'gradle.plugin.com.auth0.gradle:oss-library:0.10.0'
-    }
-}
-
-repositories {
-    jcenter()
-}
-
 dependencies {
```

Comment:
The addition of these dependencies aren't new dependencies, they are just explicitly added now. Gradle 4 was allowing the usage of these transient dependencies through other dependencies, however when updating to Gradle 6 it did not. I did consider not adding the commons-codec dependency (used for Base64 encoding), and instead using the built-in Java 8 functionality, but decided that's something that should be done in a separate PR so we can test/verify.